pdns/lua-recursor4.cc

```diff
@@ -159,6 +159,21 @@ static int getFakePTRRecords(const DNSName& qname, const std::string& prefix, ve
 
 }
 
+vector<string> RecursorLua4::DNSQuestion::getEDNSFlags() const
+{
+  vector<string> ret;
+  if (*ednsFlags & EDNSOpts::DNSSECOK)
```

Comment:
It would be safer to check whether `ednsFlags` is set before de-referencing it. It's not set in the `preoutquery` hook for example.